Specs/Scene/ModelSpec.js

```diff
@@ -1468,6 +1410,46 @@ defineSuite([
         });
     });
 
+    it('Loads with incrementallyLoadTextures set to true', function() {
+        return loadModelJson(texturedBoxModel.gltf, {
+            incrementallyLoadTextures : true
+        }).then(function(m) {
+            // Get the rendered color of the model before textures are loaded
+            var loadedColor = verifyRender(m);
+
+            pollToPromise(function() {
+                // Render scene to progressively load textures
+                scene.renderForSpecs();
+                // Textures have finished loading
+                return !defined(m._loadResources);
```

Comment:
I would like some advice for this area. In order to check that the model is done loading textures I used this line, but it's not very obvious. Maybe we can have a flag in `Model` that says if textures are loaded.